aicspylibczi/CziFile.py

```diff
@@ -459,16 +459,16 @@ def meta(self):
 
         Returns
         -------
-        str
-            An lxml.etree of the metadata as a string
+        xml.etree.ElementTree.Element
+            The root element of the metadata tree
 
         """
         if self.meta_root is None:
             meta_str = self.reader.read_meta()
-            self.meta_root = etree.fromstring(meta_str)
+            self.meta_root = ET.fromstring(meta_str)
 
         if self.metafile_out:
-            metastr = etree.tostring(self.meta_root, pretty_print=True).decode("utf-8")
+            metastr = ET.tostring(self.meta_root).decode("utf-8")
             with open(self.metafile_out, "w") as file:
                 file.write(metastr)
```

Comment:
I am a bit confused as to what is happening here. Why are you writing any files???

Reply:
The aicspylibczi has always had an option to dump the metadata to a provided file. In the context of aicsimageio it's irrelevant and in the greater context it's likely not used much but one could do `CziFile("in.czi", metafile_out="txt.xml").meta` and have it dump the XML to txt.xml.

Reply:
Interesting.... Alright :shrug: